aztk/node_scripts/setup_host.sh

```diff
@@ -2,19 +2,21 @@
 
 # Entry point for the start task. It will install all dependencies and start docker.
 # Usage:
-# setup_node.sh [container_name] [gpu_enabled] [docker_repo] [docker_cmd]
+# setup_host.sh [container_name] [gpu_enabled] [docker_repo] [docker_cmd]
```

Comment:
This needs to be updated to `#setup_host.sh [container_name] [repo_name]`. I keeping docker in the name of the variable for the docker repo is also easier to read. Maybe `docker_repo_name` is best?